store/storetest/emoji_store.go

```diff
@@ -30,33 +30,26 @@ func testEmojiSaveDelete(t *testing.T, ss store.Store) {
 		Name:      model.NewId(),
 	}
 
-	if _, err := ss.Emoji().Save(emoji1); err != nil {
-		t.Fatal(err)
-	}
+	_, err := ss.Emoji().Save(emoji1)
+	require.Nil(t, err, err)
 
-	if len(emoji1.Id) != 26 {
-		t.Fatal("should've set id for emoji")
-	}
+	require.Equal(t, len(emoji1.Id), 26, "should've set id for emoji")
 
 	emoji2 := model.Emoji{
 		CreatorId: model.NewId(),
 		Name:      emoji1.Name,
 	}
-	if _, err := ss.Emoji().Save(&emoji2); err == nil {
-		t.Fatal("shouldn't be able to save emoji with duplicate name")
-	}
+	_, err = ss.Emoji().Save(&emoji2)
+	require.NotNil(t, err, "shouldn't be able to save emoji with duplicate name")
 
-	if err := ss.Emoji().Delete(emoji1, time.Now().Unix()); err != nil {
-		t.Fatal(err)
-	}
+	err = ss.Emoji().Delete(emoji1, time.Now().Unix())
+	require.Nil(t, err, err)
 
-	if _, err := ss.Emoji().Save(&emoji2); err != nil {
-		t.Fatal("should be able to save emoji with duplicate name now that original has been deleted", err)
-	}
+	_, err = ss.Emoji().Save(&emoji2)
+	require.Nilf(t, err, "%v, should be able to save emoji with duplicate name now that original has been deleted", err)
```

Comment:
```suggestion
	require.Nil(t, err, "should be able to save emoji with duplicate name now that original has been deleted")
```

Reply:
We were logging error earlier, hence kept this. Do we not want to log the error?